packages/client/components/ScopingSearchResultItem.tsx

```diff
@@ -66,7 +66,10 @@ const ScopingSearchResultItem = (props: Props) => {
   const {onCompleted, onError, submitMutation} = useMutationProps()
   const disabled = !isSelected && usedServiceTaskIds.size >= Threshold.MAX_POKER_STORIES
   const isTemp = isTempId(serviceTaskId)
-  const onClick = () => {
+
+  const handleClick = (e: React.MouseEvent) => {
+    e.stopPropagation()
```

Comment:
-1 We have quite a few places where we stop event propagation in the app and to me it's never clear why it was put there, so it's impossible to remove or retest. Please add a comment.